.pre-commit-config.yaml

```diff
@@ -1,12 +1,29 @@
 repos:
--   repo: https://github.com/python/black
-    rev: 20.8b1
-    hooks:
-    - id: black
-      pass_filenames: true
```

Comment:
this is the default option for `pass_filenames` https://pre-commit.com/#creating-new-hooks#creating-new-hooks